app/api/route/error_handlers.py

```diff
@@ -1,166 +1,27 @@
-
-    # These are all handled with the same generic exception handler to make them uniform in structure.
-    connexion_app.add_error_handler(NotFound, http_exception_handler)
-    connexion_app.add_error_handler(HTTPException, http_exception_handler)
-    connexion_app.add_error_handler(Forbidden, http_exception_handler)
-    connexion_app.add_error_handler(Unauthorized, http_exception_handler)
-
-    # Override the default internal server error handler to prevent Flask
-    # from using logging.error with a generic message. We want to log
-    # the original exception.
-    #
-    # We handle all 500s here but only expect InternalServerError instances,
-    # as indicated by the documentation. Calling out InternalServerError explicitly
-    # here would not override the default internal server error handler.
-    #
-    connexion_app.add_error_handler(500, internal_server_error_handler)
-
-
-def http_exception_handler(http_exception: HTTPException) -> Response:
-    return error_response(
-        status_code=http_exception, message=str(http_exception.description), errors=[]
-    ).to_api_response()
-
-
-def internal_server_error_handler(error: InternalServerError) -> Response:
-    # Use the original exception if it exists.
-    exception = error.original_exception or error
-
-    logger.exception(str(exception), extra={"error.class": type(exception).__name__})
-
-    return http_exception_handler(error)
-
-
-def validation_request_handler(validation_exception: ValidationException) -> Response:
-    for error in validation_exception.errors:
-        log_validation_error(validation_exception, error, is_unexpected_validation_error)
-
-    return error_response(
-        status_code=BadRequest,
-        message=validation_exception.message,
-        errors=validation_exception.errors,
-        data=validation_exception.data,
-    ).to_api_response()
-
-
-def handle_pydantic_validation_error(exception: pydantic.ValidationError) -> Response:
-    return validation_request_handler(convert_pydantic_error_to_validation_exception(exception))
-
-
-# Some pydantic errors aren't of a format we like
-pydantic_error_type_map = {"value_error.date": "date"}
-
-
-def convert_pydantic_error_to_validation_exception(
-    exception: pydantic.ValidationError,
-) -> ValidationException:
-    errors = []
-
-    for e in exception.errors():
-        err_type = e["type"]
-        if err_type in pydantic_error_type_map:
-            err_type = pydantic_error_type_map[err_type]
-
-        err_field = e["loc"][0]
-        err_message = e["msg"]
-
-        errors.append(
-            ValidationErrorDetail(
-                type=err_type,
-                message=f'Error in field: "{err_field}". {err_message.capitalize()}.',
-                rule=None,
-                field=".".join(str(loc) for loc in e["loc"]),
-            )
+def add_error_handlers_to_app(app: APIFlask) -> None:
```

Comment:
APIFlask has much saner error formatting than Connexion (ie. multiple validation errors gives you all the errors). I only kept this to make clear you can adjust the formatting if desired.